utility/src/main/java/org/oppia/android/util/parser/FakeImageLoader.kt

```diff
@@ -0,0 +1,32 @@
+package org.oppia.android.util.parser
+
+import android.graphics.Bitmap
+import android.graphics.drawable.PictureDrawable
+import android.widget.ImageView
+import javax.inject.Inject
+
+/** A [FakeImageLoader] that is used in tests. */
```

Comment:
Nice idea! This isn't quite a full-fledged fake since it's not fulfilling the entire contract of ``ImageLoader``, though. Could we maybe make this a 'TestGlideImageLoader' rather than a fake & actually use the real GlideImageLoader except for ``loadDrawable``? Then we're effectively proxying Glide but still leveraging it except when we can't. Longer term we can move toward a proper fake (e.g. by loading a fake bitmap or svg instead of the real one), but this at least keeps the behavior in parity with production when operating unit tests.

Reply:
Ok, so now `TestGlideImageLoader` extends `GlideImageLoader`, and for that I had to `open` `GlideImageLoader`.